src/components/ProjectTagElem.tsx

```diff
@@ -0,0 +1,32 @@
+import { CloseOutlined } from '@ant-design/icons'
+import { ProjectTag } from 'models/project-tags'
+import { twMerge } from 'tailwind-merge'
+
+export default function ProjectTagElem({
+  tag,
+  className,
+  selected,
+  onClick,
+}: {
+  tag: ProjectTag
+  className?: string
+  selected?: boolean
+  onClick?: (tag: ProjectTag) => void
+}) {
+  return (
+    <div
+      onClick={() => onClick?.(tag)}
+      className={twMerge(
+        `cursor-pointer rounded-full py-1 px-3 ${
+          selected
+            ? 'bg-smoke-300 font-medium dark:bg-slate-400'
+            : 'bg-smoke-100 dark:bg-slate-600'
+        }`,
```

Comment:
**suggestion (non-blocking)**, i think twMerge takes arbitrary number of args, so instead of a template literal, this could just be another arg. Not important!